eZ/Bundle/EzPublishRestBundle/Tests/Functional/RoleTest.php

```diff
@@ -633,4 +676,52 @@ private function roleDraftHrefToRoleHref($roleDraftHref)
     {
         return str_replace('/draft', '', $roleDraftHref);
     }
+
+    /**
+     * Creates and published a role with $identifier.
```

Comment:
Typo, as a matter of fact. Don't even check `d` and `s` _are_ next to each other (http://bash.org/?5300).